### ISSUE TITLE
Healthcare Widgets- Render API error message when MHV API fails instead of hanging/spinning

### ISSUE THREAD

**Issue by chriskim2311** [department-of-veterans-affairs/va.gov-cms#15205]
STATUS: Wes posed clarifying questions in the spike ticket

### User story

**AS A** Veteran
**I WANT** see a message(warning or error) when the MHV API has failed for the Cerner or healthcare widgets
**SO THAT** I have a better understanding of what is impacting my experience if something has gone wrong.

### Engineering notes / background
[Spike results](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13062#issuecomment-1712051914)

Recommend adding try/catch block to unauthenticated widget(cta-widget). 
In the `componentDidMount()` method we want to prevent `mhv_account` API failure to result in a broken link or invalid loading state. 

Implement same warning message as implemented in Cerner widget. 

### Quality / testing notes
*Test across multiple test users and verify that widgets are not affected. - Test User Database ones.
*Force a failed API state to check the warning message that is rendered locally. 

## Acceptance criteria
- [ ] Verified that if MHV API fails, error message will display, same as Cerner does today.
- [ ] Verified the negative case, i.e., that current functionality of widgets is not affected when services are responding

**Comment by chriskim2311** [comment]
@randimays Yup! I was using test account users that were associated with Cerner and ones that weren't

**Comment by FranECross** [comment]
Check with  Dave on ownership of the api/widget

**Comment by FranECross** [comment]
[11/20/23] Dave has confirmed we should transition to the team that now owns these features.

**Comment by jilladams** [comment]
Closing. PW no longer owns the CTA widgets on these pages.
